contents/handbook/company/contributing-to-product-design.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Contributing to product design
+sidebar: Handbook
+showTitle: true
+---
+
+## Getting started
+* Design work happens in [figma](https://figma.com/) in the [App project](https://www.figma.com/files/project/17131046/App?fuid=987006441684352800). If you need access to a figma project, ask [Cory Watilo](/handbook/company/team/#cory-watilo-lead-designer) or [Chris Clark](/handbook/company/team#chris-clark-product-designer).
+* All product design work uses the [Inter font](https://fonts.google.com/specimen/Inter).
+* Product does not have a design system at this time. There is some component documentation in progress in the [design system project](https://www.figma.com/file/Y9G24U4r04nEjIDGIEGuKI/PostHog-Design-System-One?node-id=576%3A308).
+
+## Reviews &amp; critique
+
+In order to ensure users have a good experience, we review each others' work. When requesting feedback, be sure to consider the following guidelines:
+* What kind of feedback are you looking for? Be as specific as possible.
+* Have you provided sufficient context for the solution you're proposing? It can be hard to measure the success of a solution if you aren't fully aware of the problem being solved. For any work we're reviewing, there should be a corresponding github issue.
+* Does your design use existing components? If not, do you have some justification for why you've deviated from the current patterns?
+* What are the advantages of this component over alternative components?
+* Is your design consistent with the visual style we have established for the product? If you're deviating, be sure to provide some details on why and on what problems you're hoping to solve.
+* Have you considered pseudo states and other interaction patterns for your designs? Hover, focus, animation, etc.
+* Does your design follow basic UX best practices?
+* Does your design use clear language, correct grammar, and adhere to our [writing style](/handbook/company/communication#writing-style)?
```

Comment:
When should these guidelines apply? I think there are cases where making quick mockups is the best approach. Example: If you're blocked by design and you're just putting something together for a new feature that will later go through design lifecycle.

Reply:
These guidelines apply to any design that is being considered for implementation and use by the majority of users - note that experiments and isolated alpha features in production aren't really part of this. Sometimes we iterate quickly in code depending on the type of problem being solved. But if we are putting together a polished, complete, swiss watch-like experience, then this is the stuff to consider before hand off.